src/OrchardCore.Modules/OrchardCore.Autoroute/Handlers/AutoroutePartHandler.cs

```diff
@@ -126,6 +129,12 @@ public override async Task UpdatedAsync(UpdateContentContext context, AutorouteP
                     ContentItem = part.ContentItem
                 };
 
+                var localizationPart = part.ContentItem.As<LocalizationPart>();
```

Comment:
It's better to use the `GetContentCultureAsync` method which uses the `CultureAspect1`. Such that it will work without a part that is not `LocalizationPart`.

Reply:
ok for that

Reply:
> It's better to use the `GetContentCultureAsync` method

I was thinking about this before I started on this PR, but I'm not sure if access the required services from `LiquidViewTemplate` is the right way